src/lib/encryption/encryptData.ts

```diff
@@ -0,0 +1,40 @@
+import crypto from "crypto";
+
+const algorithm = "aes-256-cbc";
+const key = process.env.NEXT_PUBLIC_ENCRYPTION_KEY;
+const iv = crypto.randomBytes(16); // generate a random initialization vector
+
+if (!key) {
+  throw new Error("ENCRYPTION_KEY is not set");
```

Comment:
could be a good idea to verify the length of the key here to make sure it's configured correctly.